targets/extension

```diff
@@ -29,12 +29,33 @@ if [ "$XMETHOD" = 'x11' ]; then
 fi
 
 # Use croutonurlhandler as a fallback URL handler
-handler='/usr/local/bin/croutonurlhandler'
-for link in x-www-browser gnome-www-browser; do
-    if ! update-alternatives --query "$link" | grep -q "$handler"; then
-        update-alternatives --install "/usr/bin/$link" "$link" "$handler" 15
-    fi
-done
+if [ "${DISTROAKA:-"$DISTRO"}" = 'debian' ]; then
+    handler='/usr/local/bin/croutonurlhandler'
+    for link in x-www-browser gnome-www-browser; do
+        if ! update-alternatives --query "$link" | grep -q "$handler"; then
+            update-alternatives --install "/usr/bin/$link" "$link" "$handler" 15
+        fi
+    done
+elif [ "${DISTROAKA:-"$DISTRO"}" = 'arch' ]; then
+    install arch=desktop-file-utils,
+
+    cat > '/usr/local/share/applications/crouton.desktop' <<END
```

Comment:
> Do you need to ensure the directory exists first?

Yes. Fixed.

> We probably want to do this on Debian as well.

I did it that way because `update-alternatives` does not exist on Debian. But, yeah, I guess we could use that approach on Debian too. I'll add a FIXME in the code, for now.